pmi.md

```diff
@@ -0,0 +1,161 @@
+
+```python
+from database.db import db
+
+class PMI(db.Document):
+    downpayment_percentage = db.IntField(required=True, unique=True)
+    620_639 = db.FloatField(required=True)
+    640_659 = db.FloatField(required=True)
+    660_679 = db.FloatField(required=True)
+    680_699 = db.FloatField(required=True)
+    700_719 = db.FloatField(required=True)
+    720_739 = db.FloatField(required=True)
+    740_759 = db.FloatField(required=True)
+    760_850 = db.FloatField(required=True)
+```
+
```

Comment:
I should probably delete this, but I'm going to wait until this is transitioned to our ReadMe.md

Reply:
Please don't delete until you are comfortable this information lives somewhere permanent or all BEnders fully understand it 😺